hw/ip/lc_ctrl/data/lc_ctrl.hjson

```diff
@@ -32,25 +32,71 @@
       default: "4",
       local:   "true"
     }
-    { name:    "NumLcStateBits",
+    { name:    "CsrLcStateWidth",
       desc:    "Number of life cycle state enum bits.",
       type:    "int"
       default: "4",
       local:   "true"
     }
-    { name:    "NumLcCntBits",
-      desc:    "Number of life cycle state enum bits.",
+    { name:    "CsrLcCountWidth",
+      desc:    "Number of life cycle transition counter bits.",
       type:    "int"
       default: "5",
       local:   "true"
     }
+    { name:    "CsrLcIdStateWidth",
+      desc:    "Number of life cycle id state enum bits.",
+      type:    "int"
+      default: "2",
+      local:   "true"
+    }
   ]
 
   /////////////////////////////
   // Intermodule Connections //
   /////////////////////////////
 
   inter_signal_list: [
+    // life cycle JTAG TAP
+    { struct:  "jtag"
+      type:    "req_rsp"
+      name:    "jtag"
+      act:     "rsp"
+      package: "rv_dm_pkg"
+    }
+    // Scanmode enable
```

Comment:
i think scanmode is actually special cased in our connections.
In your hjson if you declare you have scan, it automatically creates that.
see https://github.com/lowRISC/opentitan/blob/master/hw/ip/spi_device/data/spi_device.hjson#L23

Reply:
although we could argue it's better to make this more generic, but it's what we have for now.

Reply:
oh I see. let's just use that special key then.